evap/staff/importers/json.py

```diff
@@ -410,27 +461,32 @@ def _import_evaluation(self, course: Course, data: ImportEvent) -> Evaluation:
             self.statistics.warnings.append(
                 WarningMessage(
                     obj=data["title"],
-                    message=f"Event has an unknown language: {data['language']}, main language has been set to undecided",
+                    message=f"Event has an unknown language (\"{data['language']}\"), main language was set to undecided",
                 )
             )
 
         participants = self._get_user_profiles(data["students"]) if "students" in data else []
 
         defaults = {
-            "name_de": self._clean_whitespaces(name_de),
-            "name_en": self._clean_whitespaces(name_en),
+            "exam_type": exam_type,
             "vote_start_datetime": evaluation_start_datetime,
             "vote_end_date": evaluation_end_date,
             "wait_for_grade_upload_before_publishing": wait_for_grade_upload_before_publishing,
             "weight": weight,
             "is_rewarded": is_rewarded,
             "main_language": main_language,
         }
-        evaluation, created = Evaluation.objects.get_or_create(
-            course=course,
-            cms_id=data["gguid"],
-            defaults=defaults,
-        )
+
+        created = not evaluation
+        if not evaluation:
+            evaluation = Evaluation.objects.create(
+                course=course,
+                cms_id=data["gguid"],
+                name_de=name_de,
+                name_en=name_en,
```

Comment:
Why are these not in `defaults`? Do we not want to log in some case? Please add a comment :)